templates/patient/card/erx.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends "patient/card/card_base.html.twig" %}
+
+{% block content %}
+<div class="list-group list-group-flush">
+{% if rxList|length > 0 %}
+
+    {% for rx in rxList %}
+
+        {% if rx.size > 0 %}
+            {% set unit = " (" ~ rx.size|text ~ " " ~ rx.unit ~ ") " %}
```

Comment:
`rx.unit|text`

Reply:
actually if this is not being echoed here yet, then wouldn't escape this stuff here and would instead do it when it is echoed below